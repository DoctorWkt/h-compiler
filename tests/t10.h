int c[]= { 1,3,5,7,9 };
void main()
{
  int x;
  int y=3;
  int z=4;

  x= y + z;
  printn(x);
  x= c[0] + y;
  printn(x);
  x= y + c[2];
  printn(x);
  x= c[1] + c[2];
  printn(x);
  x= c[2] + c[3] + c[4];
  printn(x);
}

