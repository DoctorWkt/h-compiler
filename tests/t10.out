000007
000004
000010
000010
000025
